Added an entry for linux-meta and linux-signed (#16)
canonical/kernel-docs: docs/reference/glossary.md

  ABI remains unchanged.

linux-meta
  Refers to a set of meta-packages in Linux distributions like Ubuntu. These meta-packages 
  do not contain the kernel binaries or source code themselves but instead define dependencies

>   that point to the latest kernel packages. By installing a linux-meta package (e.g., linux-meta-generic), users can ensure they always receive the latest version of a specific kernel series through updates.

Reviewer: ```suggestion
  that point to the latest kernel packages. By installing a linux-meta package (e.g., linux-generic), users can ensure they always receive the latest version of a specific kernel series through updates.
```
`linux-generic` is the name of one of the packages constructed from the linux-meta source package, so I'd use it as an example here instead.

  In the kernel development and Stable Release Update (SRU) lifecycle, linux-meta acts as a bridge between the release of new kernel versions and the package manager. When a new kernel version is released and marked stable, the linux-meta package is updated to reference the new version, allowing automatic upgrades.

linux-signed 

>   Refers to kernel packages that are cryptographically signed to ensure their integrity and authenticity. These signatures are crucial for secure boot environments, as they enable the system firmware to verify that the kernel has not been tampered with and is from a trusted source. In the kernel SRU lifecycle, linux-signed is created after the corresponding unsigned kernel (e.g., linux-image) has been built. The signing process is part of the release pipeline, ensuring compliance with secure boot requirements and enhancing security in the kernel deployment process. This package works in tandem with the linux-meta package to deliver signed kernel updates.

Reviewer: Maybe instead of `linux-image`, it'd be better to use a full name, such as `linux-image-unsigned-6.8.0-50-generic`. There are other `linux-image*` packages, such as `linux-image-6.8.0-50-generic`, which are signed.

Author: Thank you for the feedback, I will do so.

Reviewer 2: Thanks @magalilemes , I agree. A real example of the package name is helpful at each point.

  ABI remains unchanged.

linux-meta
  Refers to a set of meta-packages in Linux distributions like Ubuntu. These meta-packages do not 
  contain the kernel binaries or source code themselves but instead define dependencies that point 
  to the latest kernel packages. By installing a linux-meta package (e.g. linux-generic), users can 
  ensure they always receive the latest version of a specific kernel series through updates. In the 
  kernel development and {term}`SRU` lifecycle, linux-meta acts as a bridge between the release of new 

>   kernel versions and the package manager. When a new kernel version is releasedand marked stable, the linux-meta package is updated to reference the new version, allowing automatic upgrades.

Reviewer 2: Max line length here should be consistent with previous line.

Thanks.

linux-signed 

>   Refers to kernel packages that are cryptographically signed to ensure their integrity and authenticity. These signatures are crucial for secure boot environments, as they enable the system firmware to verify that the kernel has not been tampered with and is from a trusted source. In the kernel {term}`SRU` lifecycle, linux-signed is created after the corresponding unsigned kernel (e.g. linux-image-unsigned-6.8.0-50-generic) has been built. The signing process is part of the release pipeline, ensuring compliance with secure boot requirements and enhancing security in the kernel deployment process. This package works in tandem with the linux-meta package to deliver signed kernel updates.

Reviewer 2: Max line length / char count each line here should be consistent.
80 char limit is pretty standard :) 

Thanks.

  ABI remains unchanged.

linux-meta
  Refers to a set of meta-packages in Linux distributions like Ubuntu. These meta-packages 
  do not contain the kernel binaries or source code themselves but instead define dependencies

>   that point to the latest kernel packages.By installing a linux-meta package (e.g., linux-generic), 

Reviewer 2: ```suggestion
  that point to the latest kernel packages. By installing a linux-meta package (e.g., linux-generic), 
```

Missing whitespace.

  users can ensure they always receive the latest version of a specific kernel series through updates.

>   In the kernel development and Stable Release Update (SRU) lifecycle, linux-meta acts as a bridge between

Reviewer 2: ```suggestion
  In the kernel development and {term}`SRU` lifecycle, linux-meta acts as a bridge between
```

Thanks for adding SRU as a glossary term also, @Jetoson . And since there's a glossary term available I suggest just skipping the part when SRU is expanded here and just link to it with the `term` directive.

  the release of new kernel versions and the package manager. When a new kernel version is released and
  marked stable, the linux-meta package is updated to reference the new version, allowing automatic upgrades.

linux-signed 
  Refers to kernel packages that are cryptographically signed to ensure their integrity and authenticity.
  These signatures are crucial for secure boot environments, as they enable the system firmware to verify

>   that the kernel has not been tampered with and is from a trusted source. In the kernel SRU lifecycle,

Reviewer 2: ```suggestion
  that the kernel has not been tampered with and is from a trusted source. In the kernel {term}`SRU` lifecycle,
```

Similarly link directly to SRU glossary term.

>   linux-signed is created after the corresponding unsigned kernel (e.g., linux-image-unsigned-6.8.0-50-generic)

Reviewer 2: ```suggestion
  linux-signed is created after the corresponding unsigned kernel (e.g. linux-image-unsigned-6.8.0-50-generic)
```

This is a style preference but a "," isn't necessary after the "e.g.". Thanks!
